src/styles/blocks/separator.scss

```diff
@@ -0,0 +1,16 @@
+@use "../utils/mixins" as *;
+
+.separator {
+  width: 100vw;
+  height: 200px;
+  background-image: url(./images/separator_image.jpg);
+  background-size: cover;
+  background-position: bottom, center;
```

Comment:
The background-position property value should be 'center bottom' instead of 'bottom, center'. This property doesn't accept comma-separated values.